cmd/server/flags.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"flag"
+	"os"
+)
+
+type ServerConfig struct {
+	Endpoint string
+}
+
+func NewConfig() ServerConfig {
+	endpointFlag := ""
+	flag.StringVar(&endpointFlag, "a", "localhost:8080", "server endpoint")
+	flag.Parse()
+
+	if endpointEnv := os.Getenv("ADDRESS"); endpointEnv != "" {
```

Comment:
Как улучшение можно добавить префикс приложения к переменным окружения во избежание коллизии с другими переменными окружения. 

```suggestion
	if endpointEnv := os.Getenv("METRICS_SERVER_ADDRESS"); endpointEnv != "" {
```